docs/ui/index.rst

```diff
@@ -0,0 +1,59 @@
+.. _ref_ui_overview:
+
+==
+UI
+==
+
+EdgeDB UI is a beautiful, feature-rich admin panel baked directly into all 
+EdgeDB instances.
+
+The UI automatically opens in your default browser on a ``localhost`` page
+via a single command:
+
+.. code-block:: bash
+
+  $ edgedb ui
+
+Alternatively, you can print a url to paste into the browser instead of
+automatically opening the UI:
+
+.. code-block:: bash
+
+  $ edgedb ui --print-url
+
+The UI interface for :ref:`EdgeDB Cloud <ref_cli_edgedb_cloud>` is nearly
+identical to the standard EdgeDB UI, aside from extra pages on cloud-related
+items such as org settings, billing, and usage metrics.
+
+The command to open the EdgeDB UI is a CLI command, documented
+:ref:`here <ref_cli_edgedb_ui>`.
+
+The UI is served by default by development instances. To enable the UI on a
+production instance, use the ``--admin-ui`` option with ``edgedb-server``
+or set the ``EDGEDB_SERVER_ADMIN_UI`` :ref:`environment variable
+<ref_reference_envvar_admin_ui>` to ``enabled``.
+
+The UI has three similar pages that each allow users to query the database:
+the REPL, the editor, and the data explorer.
+
+Outright new users to EdgeDB who need to query or work with objects in a
+database will find the data explorer tab the easiest to use, as it allows
+simple point-and-click access to objects without using any EdgeQL. The next
+easiest page to use is the query builder inside the Editor tab, which does
+use EdgeQL but is also point-and-click as it walks users through every step
+of a query. The REPL is the most advanced tab in the UI as it requires the
+user to be able to compose queries in EdgeQL.
```

Comment:
Aside from ease of use, would it make sense to enumerate some common use-cases for each tool?

Reply:
Gave your idea a try by rewriting this part based on use cases. What do you think?

https://github.com/edgedb/edgedb/commit/461890dfe8b5e10aa6123de65314d58d6771545d

At the end I opted to mention the CLI as another tool as I found out the other day that a surprisingly large number of people have only ever used the UI REPL. I guess my CLI-centred worldview had given me the impression that most people used it. It's less flashy but I'd like UI users to know that it's an option for a tool that gives a more instantaneous response to queries.